packages/react-components/review/beta/react-components.api.md

```diff
@@ -1202,6 +1209,14 @@ export interface _Identifiers {
     videoTile: string;
 }
 
+// @beta
+export interface ImageFileMetadata extends BaseFileMetadata {
+    // (undocumented)
+    attachmentType: 'inlineImage' | 'attachedImage';
```

Comment:
Is it possible to use only one attachmentType for one type
The attachmentType property is more like a `kind` property for identify the object type, so a one to one mapping would be nice
or we use another parameter name as `kind` and this one is dedicated for attachedImage or inlineImage

Reply:
Confirmed that we can remove the `attachedImage` type since it is not being used